DispatchMayhem/Assets/Scripts/Finances.cs

```diff
@@ -1,24 +1,76 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
 
 public class Finances : MonoBehaviour
 {
+    private static Finances instance = null;
+    public static Finances inst { get { return instance; } }
+
+    public Text cashText;
+
     public float currCurrency;
     //int spentCurrency;
     //int TotalCurrencyMade = currCurency + spentCurreny;
-    
-    // Start is called before the first frame update
-    void Start()
+
+    void Awake()
     {
-        //currCurrency = 
+        if (instance == null)
+        {
+            instance = this;
+        }
+        else if (instance != this)
+        {
+            Destroy(this.gameObject);
+        }
+
+        if (PlayerPrefs.HasKey("money"))
+        {
+            currCurrency = PlayerPrefs.GetFloat("money");
+        }
+        else
+        {
+            PlayerPrefs.SetFloat("money", 1500.0f);
+            currCurrency = 1500.0f;
+        }
+
     }
+        // Start is called before the first frame update
+    //void Start()
+    //{
+
+    //}
 
     // Update is called once per frame
-    void Update()
+        void Update()
+        {
+        int tmpcur = (int)currCurrency;
+        cashText.text = tmpcur.ToString();
+        }
+
+    private void OnApplicationQuit()
     {
-        
+        PlayerPrefs.SetFloat("money", currCurrency);
```

Comment:
you may want to consider setting this playerpref anytime the user modifies their money. if they don't exit cleanly by quitting it won't save (i.e crash, force close with swipe, etc...)